addons/main/functions/UnitAction/fnc_doHide.sqf

```diff
@@ -21,10 +21,9 @@ params ["_unit", "_pos", ["_range", 55], ["_buildings", []]];
 
 // stopped -- exit
 if (
-    stopped _unit
```

Comment:
Stopped is a legacy feature from older versions. https://community.bistudio.com/wiki/stop . I thought it caught "doStop" but it didn't. Only place I've seen it use is in the Civilian FSM script.